CONTRIBUTING.md

```diff
@@ -194,6 +194,16 @@ MIRIFLAGS="-Zmiri-disable-isolation -Zmiri-tag-raw-pointers" \
     cargo +nightly miri test --features full --lib
 ```
 
+You can also perform spell-check on tokio codebase. For details of how to use the spellcheck tool, feel free to visit
+https://github.com/drahnr/cargo-spellcheck
+```
+# First install the spell-check plugin
+cargo install --locked cargo-spellcheck
+
+# Then run the cargo spell check command
+cargo spell check
+```
```

Comment:
I'd add a section for it. Also, please explain what to do if `cargo-spellcheck` rejects a word: 
If it's code related, it should be enclosed with backticks. If not, they should add it to the `.dic` file.

Reply:
Also the command should be:
```
cargo spellcheck check
```